pkg/private/xtest/graphupdater/main.go

```diff
@@ -26,12 +26,15 @@ var (
 	graphFile = flag.String("graphFile", "", "")
 	descName  = flag.String("descName", "", "")
 	linksFile = flag.String("linksFile", "", "")
+	ifIDsFile = flag.String("ifidsFile", "", "")
 )
 
 func main() {
 	flag.Parse()
 	if *linksFile != "" {
 		writeLinksToFile()
+	} else if *ifIDsFile != "" {
+		writeIfIDsToFile()
 	} else {
 		writeGraphToFile()
 	}
```

Comment:
```suggestion
	switch {
    case *linksFile != "":
		writeLinksToFile()
	case *ifIDsFile != "":
		writeIfIDsToFile()
	default:
		writeGraphToFile()
	}
```

nit: a switch is a bit more readable IMHO, but it doesn't matter, pick the one you like more.

Reply:
Done.
<!-- Reviewable comment -OQqvGrd-rurCQmBN4of:b-896fix -->
<!-- Sent from Reviewable.io -->